weave-js/src/components/PagePanelComponents/Home/Browse3/pages/CallPage/OverflowMenu.tsx

```diff
@@ -67,7 +92,7 @@ export const OverflowMenu: FC<{
             style={{marginLeft: '4px'}}
           />
         }
-        offset="-78px, -16px"
+        offset="-178px, -16px"
```

Comment:
intentional?

Reply:
yes,the menu width increased and so must the offset